homeassistant/components/igloohome/strings.json

```diff
@@ -0,0 +1,26 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Add API Access Credentials",
+        "description": "Set your account's API keys to enable Home Assistant to poll the status of your devices.",
```

Comment:
I'd remove the title and set the description to something more helpful like where we can find those API keys for example